### PR TITLE
Refactor quartz-cluster module and added Native test

### PR DESCRIPTION
**Pull request by Sgitario**
- Usage of start() method from QuarkusProdModeTest
- Add native test for quartz-cluster

### CODE REVIEW

File: 017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java [hunk 1]
```diff
@@ -50,9 +53,8 @@ private void whenBothNodesAreUpAndRunning() {
         assertFalse(nodeTwoApp.getStartupConsoleOutput().isEmpty(), "Node Two should be up and running");
     }
 
-    private void whenShutdownNodeOne() throws Exception {
-        // TODO: Workaround for https://github.com/quarkusio/quarkus/issues/14961
-        nodeOneApp.afterAll(null);
+    private void whenShutdownNodeOne() {
+        nodeOneApp.stop();
```
Comment (rsvoboda) [inline 017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java:57]: https://github.com/quarkusio/quarkus/issues/14961 is fixed for 1.13, not 1.11.

We still primarily care about 1.11 ...

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: 017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java (hunk 1 above)
Comment (Sgitario) [reply 017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java:57]: This change is already in another module in master.
I think the way to go is to disable these modules in master?

Comment (rsvoboda) [reply 017-quartz-cluster/src/test/java/io/quarkus/qe/quartz/AnnotationScheduledJobsQuartzTest.java:57]: ah, I guess you mean 018-quarkus-runtime-properties ++ 019-quarkus-consul ...

Let's keep it in master, adjustments need to be handled in 1.11 branch PR

**Review by rsvoboda — APPROVED** [review]
Approved, but let's wait with merging before https://github.com/quarkus-qe/beefy-scenarios/pull/100 gets merged

**Pull request merged**